lua/telescope/pickers/layout_strategies.lua

```diff
@@ -414,7 +554,73 @@ layout_strategies.current_buffer = function(self, _, _)
     results = results,
     prompt = prompt,
   }
-end
+end)
+
+--- Bottom pane can be used to create layouts similar to "ivy".
+---
+--- For an easy ivy configuration, see |themes.get_ivy()|
+layout_strategies.bottom_pane = make_documented_layout('bottom_pane', vim.tbl_extend("error", shared_options, {
```

Comment:
`bottom_pane` exists twice. They look identical on first glance. But one should still be removed :)